src/app/test-record/test-record.component.ts

```diff
@@ -0,0 +1,43 @@
+import {Component, OnInit, ChangeDetectionStrategy} from '@angular/core';
+import {select, Store} from '@ngrx/store';
+import {IAppState} from '@app/technical-record/adr-details/adr-details-form/store/adrDetailsForm.state';
+import {selectSelectedVehicleTestResultModel} from '@app/store/selectors/VehicleTestResultModel.selectors';
+import {Observable} from 'rxjs';
+import {ActivatedRoute} from '@angular/router';
+
+@Component({
+  selector: 'vtm-test-record',
+  templateUrl: './test-record.component.html',
+  styleUrls: ['./test-record.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TestRecordComponent implements OnInit {
+
+  testResultObservable$: Observable<any>;
+  testType: any;
```

Comment:
specify the type to be `TestType`

Reply:
Fixed into 10283, but I can fix in here also